data_steward/gcloud/bq/__init__.py

```diff
@@ -23,6 +23,10 @@
 from constants.utils import bq as consts
 from common import JINJA_ENV
 
+tracer_provider = TracerProvider()
```

Comment:
Can you please explain why we needed to move these lines out of the class?

Reply:
If tracer provider is not set global, then the DefaultTracerProvider will provide no-op spans that don't do anything. We set it global so we don't fail to collect telemetry.